public/pages/cadastro/cadastro_pais.html

```diff
@@ -8,11 +8,11 @@
     <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
     <meta name="generator" content="Hugo 0.101.0">
     <title>Cadastro dos Pais</title>
-    <link rel= "icon" href="/public/assets/images/logo.png">
+    <link rel= "icon" href="../../../public/assets/images/logo.png">
     <link rel="canonical" href="https://getbootstrap.com/docs/5.2/examples/checkout/">
     <link rel="stylesheet" href="../../css/cadastro.css">
-    <link rel="stylesheet" href="../../css/mobile.css">
-    <link rel="shortcut icon" href="../../assets/images/logoicon.png" type="image/x-icon">
+    <link rel="stylesheet" href="../../css/mobile.css">./public/assets/images/logoicon.png
```

Comment:
Sugestão de correção:

```suggestion
    <link rel="stylesheet" href="../../../css/mobile.css"/>
```